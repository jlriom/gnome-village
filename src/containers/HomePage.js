import React from "react";
import { Link } from "react-router-dom";

const HomePage = () => {
	return (
		<div className='jumbotron'>
			<h2 className='display-4'>Welcome to Brastlewark village</h2>
			<p className='lead'>React web application demo</p>
			<hr className='my-4'></hr>
			<p>Go to About page to get started using the application</p>
			<Link to='/about' className='btn btn-primary btn-lg'>
				{" "}
				About...{" "}
			</Link>
		</div>
	);
};

export default HomePage;
