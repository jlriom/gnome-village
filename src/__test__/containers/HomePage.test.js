import React from "react";
import HomePage from "../../containers/HomePage";
import renderer from "react-test-renderer";
import { MemoryRouter } from 'react-router-dom';

describe("<HomePage />", () => {
    it("renders ok", () => {
        const tree = renderer.create(
            <MemoryRouter>
                <HomePage />
            </MemoryRouter>
        );
        expect(tree).toMatchSnapshot();
    });
});
