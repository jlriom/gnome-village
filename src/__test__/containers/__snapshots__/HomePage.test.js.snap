// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> renders ok 1`] = `
<div
  className="jumbotron"
>
  <h2
    className="display-4"
  >
    Welcome to Brastlewark village
  </h2>
  <p
    className="lead"
  >
    React web application demo
  </p>
  <hr
    className="my-4"
  />
  <p>
    Go to About page to get started using the application
  </p>
  <a
    className="btn btn-primary btn-lg"
    href="/about"
    onClick={[Function]}
  >
     
    About...
     
  </a>
</div>
`;
